### PR TITLE
Adds module upgrades for the Janiborg.

### PR DESCRIPTION
**Pull request by Firecage**
This PR adds two new upgrades for the Janiborg. Well, technically adds their first upgrades since they didn't have any module specific ones. These are the Advanced Mop and the Trash Bag of Holding.

:cl: Firecage
add: The mech fabricator can now print two new janiborg upgrades. Advanced Mop and Trash Bag of holding. These two upgrades uses the advanced robotics node.
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
Instead of making it a new node why not just add it to the Advanced Robotics Programming or whatever it's called that only had the Diamond Drill upgrade and is otherwise a dead end?

**Review by SpaceManiac — CHANGES_REQUESTED** [review]
I agree with YakumoChen here. I don't think it's a big problem if cyborg janitorial doesn't actually require human janitorial; things can always be rearranged later if it's needed. The cyborg diamond drill doesn't require the regular diamond drill.

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -193,6 +193,60 @@
...
+	name = "janitor cyborg trash bag of holding"
+	desc = "A trash bag of holding replace for the janiborgs standard trash bag."
+	icon_state = "cyborg_upgrade3"
+	require_module = 1
+	module_type = /obj/item/robot_module/janitor
+
+/obj/item/borg/upgrade/tboh/action(mob/living/silicon/robot/R)
+	. = ..()
+	if(.)
+		for(var/obj/item/storage/bag/trash/cyborg/TB in R.module)
```
Comment (YPOQ) [inline code/game/objects/items/robot/robot_upgrades.dm]: R.module only contains the not-currently-equipped modules. R.module.modules has all equipped and unequipped modules.

**Comment by Firecage** [comment]
Alright, will do so.

**Review by SpaceManiac — APPROVED** [review]

**Pull request merged**